src/hera/task.py

```diff
@@ -90,6 +92,7 @@ def __init__(
         input_artifacts: Optional[List[InputArtifact]] = None,
         output_artifacts: Optional[List[OutputArtifact]] = None,
         image: str = 'python:3.7',
+        daemon: bool = False,
```

Comment:
I was looking through the `V1alpha1Template` object and it seems the default value is `None`. What do you think about making this: 
```suggestion
        daemon: Optional[bool] = None,
```
This way Hera is (1) aligned with the SDK default value and (2) avoid potentially unexpected behavior via setting the value to `False` (I don't know what this could be, tbh, but it has happened before in my XP that setting a value actually changes behavior rather than leaving stuff to default 🤷🏻‍♂️).

Reply:
agree